app/controllers/step_by_step_pages_controller.rb

```diff
@@ -76,9 +76,12 @@ def publish
   def schedule
     set_current_page_as_step_by_step
     if request.post?
-      if @step_by_step_page.update_attributes(scheduled_at: params[:scheduled_at])
+      date_params = params[:schedule][:date].permit(:year, :month, :day)
```

Comment:
Can you add a before and after controller test to prove that `scheduled_at` is still be stored correctly.

The "before" should be added to the commit before this one and test that the scheduled_at time was being set correctly and that the state was being updated as expected before any of these changes were added.

The "after" should be added with these changes and test that these changes don't affect the `scheduled_at` being saved.